src/task1.java

```diff
@@ -0,0 +1,32 @@
+import java.lang.reflect.Parameter;
+import java.util.Scanner;
+
+public class task1 {
```

Comment:
First of all, all class names should begin with the capital letter. 
Another thing, in the homework description it was stated that everything HAS to be in one class.

Reply:
I corrected everything you said. and pushed again